README.md

```diff
@@ -132,6 +132,24 @@ Additionally, you can find details and examples of how our API is structured in
 The API Reference Guide provides examples of what information is needed for a particular request and how that information would be formatted. Using those examples, you can easily determine what methods would be necessary to include that information in a request using this SDK.
 
 
+## Create a Chase Pay Transaction
+Use this method to authorize and capture a payment using a tokenized credit card number issued by Chase Pay. Chase Pay transactions are only available to merchants using the Paymentech processor.
+
+The following information is required in the request:
+
+- The payment token,
+- The expiration date,
+- The cryptogram received from the token provider,
+- The tokenRequestorName,
+- The tokenRequestorId, and
+- The tokenRequestorEci.
+
+When using the SDK to submit Chase Pay transactions, consider the following points:
+- TokenRequesterName must be populated with  “CHASE_PAY”
+- TokenRequestorID must be populated with <tokenRequestorID> as provided by Chase Pay services for each transaction during consumer checkout
```

Comment:
\<tokenRequestorID> doesn't show up in the [README](https://github.com/AuthorizeNet/sdk-php/blob/95bc0bf66a66ab449868462f8c8810ebaad445fd/README.md)

Something inside quotes doesn't show up as is in GFM.
Either escape with \ . Or, what about showing it as `code` instead?

Reply:
Alternatively, what about something like the following bullet points?
- `tokenRequesterName` must be populated with “CHASE_PAY”
- `tokenRequestorID` must be populated with TokenRequestorID as provided by Chase Pay services for each transaction during consumer checkout